joomla/joomla-cms
# [6.2] [RFC] CMSPlugin: Subscriber Registration Checker Interface

### Summary of Changes

Adding Subscriber Registration Checker. Interface that allows Plugins to be checked before listener registration.
This allows the plugin to be registered only wheen special conditions are met, eg: only for specific application.


### Testing Instructions
Code review by maintainers.

Edit https://github.com/joomla/joomla-cms/blob/98f3ecdda6701eac91a4f072ded0de488f3a5c8a/plugins/system/guidedtours/src/Extension/GuidedTours.php#L124-L126
And add `dd('Works');` at top of this method.

Then open **Administrator page**: you should get message "works"
And then open **Site page**: the page should be opened as usual


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/272
